VERSION

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
+1 for VERSION file with benefits mentioned here https://github.com/minishift/minishift-b2d-iso/pull/16#discussion_r95108894